include/cuse_lowlevel.h

```diff
@@ -9,8 +9,8 @@
   Read example/cusexmp.c for usages.
 */
 
-#ifndef _CUSE_LOWLEVEL_H_
-#define _CUSE_LOWLEVEL_H_
+#ifndef CUSE_LOWLEVEL_H_
```

Comment:
Given the leading "_" is being removed, it might make sense to remove the trailing one as well.

I think this is a good catch though; of all the major open source projects, I could see only gcc using the correct notation.